pkg/apps/index.jsx

```diff
@@ -28,7 +28,7 @@ import { ApplicationList } from "./application-list.jsx";
 import { Application } from "./application.jsx";
 import { get_metainfo_db } from "./appstream.js";
 
-import "page.css";
+import "page.scss";
```

Comment:
This commit ("Convert page.css into scss") also seems to be relatively harmless/standalone, and we could land it separately from the more complicated changes?

Reply:
Sure - I will try to break the PR into smaller pieces.

Reply:
To clarify: I was *only* suggesting breaking this PR up *if* it will take longer as a whole to land -- but IRC conversation suggests that this is by and large ready. So no need to split it up in that case, of course.